_layouts/base.html

```diff
@@ -142,6 +142,9 @@
             <div class="grid-col-auto">
               <h3 class="usa-footer__logo-heading">U.S. General Services Administration</h3>
             </div>
+            <div class="grid-col-fill text-right">
+              <a href="https://logingov.statuspage.io/" target="_blank">System Status</a>
```

Comment:
Any strong attachment to opening in a new window?

https://www.w3.org/TR/WCAG20-TECHS/G200.html

```suggestion
              <a href="https://logingov.statuspage.io/">System Status</a>
```

Reply:
Hm, as an external link it felt like a new window made more sense, it's also what we're doing on the new brochure website: https://github.com/18F/identity-site/blob/demo/_includes/footer.html#L52. What do you think @aduth?

Reply:
My preference would be to open it in the same tab. While opening external links in a new tab is a common practice, my understanding is that it's increasingly recommended to open in the same tab whenever possible, for mobile-friendliness and [improved accessibility](https://www.w3.org/TR/WCAG20-TECHS/G200.html) as mentioned above, and to avoid [recommended accommodations](https://www.w3.org/TR/WCAG20-TECHS/G201.html) associated with opening links in a new window.

There was some related discussion about this in https://github.com/18F/identity-idp/pull/4310, where in some cases we may have legitimate reason to open in new windows, but largely in our apps like the IdP, to avoid potential in-progress form data loss.

Reply:
Fair enough, I don't have a strong preference so I'll change it. Not sure if you want to make the same change on the main login.gov site as well.

Reply:
> Fair enough, I don't have a strong preference so I'll change it. Not sure if you want to make the same change on the main login.gov site as well.

Yeah, it'd be good to be consistent. I can put together a pull request.

Reply:
Following-up: I created LG-3976 to track this, since there are _many_ (100+) instances of links on the brochure site opening in new windows, and it should ideally be addressed holistically.

Reply:
Thanks for the update! That's way more instances than I would have expected, to be honest - glad you mentioned it!